code/datums/ship.dm

```diff
@@ -326,7 +326,6 @@ var/next_ship_id
 		ship.attacking_target = chosen_target
 		SSship.broadcast_message("<span class=notice>Caution! [SSship.faction2prefix(ship)] ship ([ship.name]) locking on to [SSship.faction2prefix(ship.attacking_target)] ship ([ship.attacking_target.name]).</span>",null,ship)
 
-	message_admins("[ship.name] has entered into combat at [ship.system]! [ship.attacking_player ? "" : "Combat was not due to players!"]")
```

Comment:
I agree, the earlier behavior was just to help understand what was going on under the hood with ships due to their abstraction, and to help pinpoint erratic behavior with ships if it happens